app/src/main/java/com/aspirebudgetingmobile/aspirebudgeting/activities/Splash.java

```diff
@@ -0,0 +1,68 @@
+package com.aspirebudgetingmobile.aspirebudgeting.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.ContextCompat;
+
+import com.aspirebudgetingmobile.aspirebudgeting.R;
+import com.aspirebudgetingmobile.aspirebudgeting.utils.ObjectFactory;
+import com.aspirebudgetingmobile.aspirebudgeting.utils.SessionConfig;
+import com.aspirebudgetingmobile.aspirebudgeting.utils.UserManager;
+
+public class Splash extends AppCompatActivity {
+
+    ObjectFactory objectFactory = ObjectFactory.getInstance();
+    UserManager userManager;
+    SessionConfig sessionConfig;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_splash);
+
+        try {
+            getWindow().setNavigationBarColor(ContextCompat.getColor(Splash.this, R.color.colorPrimary));
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        sessionConfig = new SessionConfig(Splash.this);
```

Comment:
Can we use ObjectFactory to create SessionConfig? The problem with using "new" operator here is that now this class is tightly coupled with the SessionConfig and this class will not be unit testable in isolation.

Reply:
Yeah sure, I will remove the param from constructor !